cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -165,6 +166,8 @@ def gen_arg_parser(cls, args_to_parse=None):
                                   help="Namespace where CephCluster is running")
         common_group.add_argument("--rgw-pool-prefix", default="",
                                   help="RGW Pool prefix")
+        common_group.add_argument("--restricted-auth-permission",default=False,
+                                  help="Restricted cephCSIKeyrings auth permissions to specific pools and cluster")
```

Comment:
It looks like this flag explicitly requires `cephfs-metadata-pool-name` too right? So let's document it.

Reply:
Added it to the add_argument `help` args.